python/packages/nisar/products/insar/InSAR_base_writer.py

```diff
@@ -926,10 +928,11 @@ def add_identification_to_hdf5(self):
             processing_type = np.bytes_('Nominal')
         elif processing_type == 'UR':
             processing_type = np.bytes_('Urgent')
-        elif processing_type == 'OD':
-            processing_type = np.bytes_('Custom')
         else:
-            processing_type = np.bytes_('Undefined')
+            processing_type = np.bytes_('Custom')
+            if processing_type != 'OD':
```

Comment:
Please take a look at the log message @gshiroma used in https://github.com/isce-framework/isce3/pull/59 . It's a bit more informative that this message, but this simpler message is also functional. Your choice if you'd like to adjust it or not. :)